app-web/src/components/Form/form.js

```diff
@@ -55,6 +55,8 @@ export const StylesWrapper = styled.div`
   input {
     display: block;
     font-size: 18px;
+    width: 500px;
+    height: 50px;
```

Comment:
this feels wrong, Why is height set? usually height is the sum of spacing and font size for input fields.